tests/README.md

```diff
@@ -51,6 +51,40 @@ bash tests/docker/ducker-ak up -j 'openjdk:11'; tests/docker/run_tests.sh
 ```
 REBUILD="t" bash tests/docker/run_tests.sh
 ```
+* Debug tests in VS Code:
+  - Run test with `--debug` flag (can be before or after file name):
```

Comment:
looks like this conflicts with existing usage running tests with debug logs (check above)
`_DUCKTAPE_OPTIONS="--debug" bash tests/docker/run_tests.sh | tee debug_logs.txt`

Can we use different option to enable debugger ex: --debugpy (or) --enable-debugger

Reply:
We can still keep this - I do need to update run_tests.sh to pass _DUCKTAPE_OPTIONS after the `--`.
My idea was that I didn't want to make sure the flags don't overlap between ducktape and ducker-ak, hence I changed ducker-ak to expect ducktape args after the `--` - like this `ducker-ak test my_test.py -- --ducktape-flag`. 
I am, however, open to simply using a different flag name - it is unlikely we'll add too many flags to the ducker-ak run command anyway, so we can simply make sure they don't match the ducktape ones. 
@omkreddy and @ewencp (with whom we had similar conversation in a different repo) - which one do you prefer?

Reply:
As in other thread, I'm fine either way. I prefer just getting things to a clean, consistent state, especially since these are tests so we don't really have the same compatibility requirements. But if that creates too much overhead/confusion (especially since we need to continue running tests on older branches), this alternative approach seems acceptable even if a bit unusual.

Reply:
@stan-confluent I am fine with current approach. Can you update `run_tests.sh` file?

Reply:
@omkreddy I've updated the run_tests.sh file to pass ducktape args after -- if those are present, can you please take another look?
I've also ran shellcheck and fixed the violations found in the changed bits of code.
Thanks!